app/Models/Campaign.php

```diff
@@ -20,7 +20,7 @@ class Campaign extends Model
      *
      * @var array
      */
-    protected $dates = ['deleted_at', 'start_date', 'end_date'];
+    protected $dates = ['start_date', 'end_date'];
```

Comment:
Since this is the first instance in the alphabetical list of file reviews, commenting here; the framework not automagically ✨ converts `deleted_at` to Carbon date instances, so no need to specify it within the `$dates` array property :)